common/test/test_sshtools.py

```diff
@@ -0,0 +1,74 @@
+# Back In Time
+# Copyright (C) 2016 Taylor Raack
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public Licensealong
+# with this program; if not, write to the Free Software Foundation,Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from contextlib import contextmanager
+import os
+import shutil
+import sys
+import tempfile
+import unittest
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+import config
+import logger
+import mount
+
+@contextmanager
+def tempdir():
+    name = tempfile.mkdtemp()
+    try:
+        yield name
+    finally:
+        shutil.rmtree(name)
+
+class TestSSH(unittest.TestCase):
+
+    def setUp(self):
+        logger.DEBUG = '-v' in sys.argv
+        self.config = config.Config()
+        self.config.set_snapshots_mode('ssh')
+        self.config.set_ssh_host('localhost')
+        self.config.set_ssh_user('traack')
```

Comment:
Decided to just use ssh keys for this. Travis builds seem to be a bit "stuck" - going to close the computer now, but maybe they'll be working for me in a few hours.